report/renderer/details-renderer.js

```diff
@@ -79,9 +79,8 @@ export class DetailsRenderer {
    * @return {Element}
    */
   _renderBytes(details) {
-    // TODO: handle displayUnit once we have something other than 'kb'
-    // Note that 'kb' is historical and actually represents KiB.
-    const value = Util.i18n.formatBytesToKiB(details.value, details.granularity);
+    // TODO: handle displayUnit once we have something other than 'KiB'
+    const value = Util.i18n.formatBytesToKiB(details.value, details.granularity || 0.1);
```

Comment:
Do we have tests for these default granularities coming out of the details renderer? If they're going to move here, something should make sure they're maintained

Reply:
added